packages/core/src/components/Tag/Tag.styles.tsx

```diff
@@ -113,44 +44,95 @@ export const StyledChip = styled(
         outline: "0 !important",
         boxShadow: "none !important",
       },
-      ...($disabled && {
-        "&:hover": {
-          backgroundColor: theme.colors.atmo3,
-        },
-        "&:focus": {
-          backgroundColor: theme.colors.atmo3,
-          outline: "none",
-          boxShadow: "none",
-          outlineOffset: 0,
-        },
-      }),
     },
-  })
-);
+  },
 
-export const StyledButton = styled(
-  forwardRef((props: HvButtonProps, ref?: PolymorphicRef<"button">) => {
-    return <HvButton {...props} ref={ref} />;
-  })
-)({
-  "& .MuiButton-startIcon": {
+  focusVisible: {},
+
+  primaryButton: {
+    background: "transparent",
+  },
+
+  label: {},
+
+  tagButton: {
     width: 16,
     height: 16,
     minWidth: 16,
     minHeight: 16,
     padding: 0,
     margin: 0,
   },
-  "& .MuiButton-textPrimary": {
-    background: "transparent",
+
+  deleteIcon: {},
+
+  disabledDeleteIcon: {
+    "&:hover": {
+      backgroundColor: theme.colors.atmo3,
+    },
+    "&:focus": {
+      backgroundColor: theme.colors.atmo3,
+      outline: "none",
+      boxShadow: "none",
+      outlineOffset: 0,
+    },
   },
-});
 
-export const StyledCloseXS = styled(
-  CloseXS,
-  transientOptions
-)(({ $color }: { $color: string }) => ({
-  "& svg .color0": {
-    fill: theme.colors[$color],
+  titleOverflow: {
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+  categorical: {
+    borderRadius: 8,
+    "&$clickable": {
+      cursor: "pointer",
+    },
+    "&:hover": {
+      borderRadius: 8,
+    },
+    "& $label": {
+      color: theme.colors.secondary,
+    },
+    "&:focus:not(:focus-visible)": {
+      outline: "0 !important",
+      boxShadow: "none !important",
+    },
   },
-}));
+  clickable: {},
+
+  categoricalFocus: {
+    "&:focus": {
+      ...outlineStyles,
+    },
+  },
+
+  disabled: {
+    backgroundColor: theme.colors.atmo3,
+    cursor: "not-allowed",
+    "& $label": {
+      color: theme.colors.secondary_60,
+    },
+  },
+
+  categoricalDisabled: {
+    backgroundColor: theme.colors.atmo3,
+    cursor: "not-allowed",
+    "& $label": {
+      color: theme.colors.secondary_60,
+    },
+    "&:hover": {
+      backgroundColor: theme.colors.atmo3,
+    },
+    "&:focus": {
+      outline: "none",
+    },
+  },
+
+  semanticTextColor: {
+    color: theme.colors.base_light,
+  },
+  categoricalTextColor: {
+    color: theme.colors.secondary,
+  },
```

Comment:
The classes `categoricalTextColor`, `semanticTextColor`, and `titleOverflow` doesn't seem to be used in the component.